src/components/va-accordion/va-accordion.tsx

```diff
@@ -42,6 +49,20 @@ export class VaAccordion {
     }
 
     const prevAttr = clickedItem.getAttribute('open') === 'true' ? true : false;
+
+    if (!this.disableAnalytics) {
+      const detail = {
+        componentName: 'Accordion',
+        action: prevAttr ? "collapse" : "expand",
+        details: {
+          header: clickedItem.header,
+          subheader: clickedItem.subeader,
```

Comment:
@bsmartin-ep Per [testing](https://github.com/department-of-veterans-affairs/va.gov-team/issues/14487#issuecomment-828479452), we'll want to update this

Reply:
Please test again @jonwehausen...